API Testing/pillar2-submission-api/99-qa/oas/X-Pillar2-Id mismatch.bru

```
@@ -0,0 +1,33 @@
+meta {
+  name: X-Pillar2-Id mismatch
+  type: http
+  seq: 2
+}
+
+get {
+  url: {{apiUrl}}/obligations-and-submissions?fromDate=2024-01-01&toDate=2024-12-310
```

```suggestion
  url: {{apiUrl}}/obligations-and-submissions?fromDate=2024-01-01&toDate=2024-12-310
```

I just spotted another `310`. Can you remove it please?

---

interesting, this and the above seem to lose sync if changed outside Bruno. Should be fixed now. Thanks.

---

interesting, this and the above seem to silently lose sync if changed outside Bruno. Should be good now. Thanks for spotting this.